openpype/hosts/houdini/api/lib.py

```diff
@@ -493,8 +501,12 @@ def reset_framerange():
     frame_start -= int(handle_start)
     frame_end += int(handle_end)
 
+    # Set frame range and FPS
+    print("Setting scene FPS to {}".format(int(fps)))
+    set_scene_fps(fps)
     hou.playbar.setFrameRange(frame_start, frame_end)
     hou.playbar.setPlaybackRange(frame_start, frame_end)
+    print("Setting current frame to {}".format(frame_start))
```

Comment:
I'd prefer printing less than more unless it's something that's extremely unreliable or just needs often debugging whether it actually did a thing - I feel like "current frame" isn't either of those things? Is there a reason you added this particular print statement?

Reply:
`hou.setFrame(frame_start)` was somehow ignored. I thought adding some delay may solve it but it didn't work, and I forgot to delete it.